plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -659,6 +688,8 @@ global proc int mayaUsdTranslatorExport (string $parent,
             $currentOptions = mayaUsdTranslatorExport_AppendFromPopup($currentOptions, "exportSkin", "skinClustersPopup");
             $currentOptions = mayaUsdTranslatorExport_AppendFromCheckbox($currentOptions, "exportBlendShapes", "exportBlendShapesCheckBox");
             $currentOptions = mayaUsdTranslatorExport_AppendFromCheckbox($currentOptions, "exportDisplayColor", "exportDisplayColorCheckBox");
+            // We currently have only one type to filter. In the future, we might need to build the array separately as more types are added.
+            $currentOptions = mayaUsdTranslatorExport_SetFromCheckbox($currentOptions, "", "filterTypes=nurbsCurve", "exportCurvesCheckBox");

```

Comment:
What you are saying here is that currently you cannot call mayaUsdTranslatorExport_SetFromCheckbox() again with a different filterType because then the option string would have two filterType args. Is that right. Instead we would need extra code here to build up the array of filter types?

Reply:
Exactly. In a future where we might add more prim filtering checkboxes, the filterType array will have to be built by reviewing all checkboxes before being assigned to the filterType export parameter.